UI/src/utils/styleRule.js

```diff
@@ -1,147 +1,180 @@
+        var exists = false;
+        for(var i=0; i < values.length ; ++i)
+          if(values[i] == propertyValue)
+            exists = true;
+        return !exists;
+      };
+      return this;
+    };
 
-      this.isBetween = function(values){
-        var expression = expressionFn[expressionFn.length-1];
-        if(!expression || expression.compare)
-          throw 'You must have on of the following functions ["where", "and", "or"] before use the "isBetween".';
-        if(values.length !== 2)
-          throw 'you must put an interval between two values [value_1, value_2]';
-        expression.compare = function(propertyValue){
-            return values[0] <= propertyValue  && propertyValue < values[1];
-        };
-        return this;
+    this.isBetween = function(values){
+      var expression = expressionFn[expressionFn.length-1];
+      if(!expression || expression.compare)
+        throw 'You must have on of the following functions ["where", "and", "or"] before use the "isBetween".';
+      if(values.length !== 2)
+        throw 'you must put an interval between two values [value_1, value_2]';
+      expression.compare = function(propertyValue){
+        return values[0] <= propertyValue  && propertyValue < values[1];
       };
+      return this;
+    };
 
-        this.use = function(obj){
-            styles.push(obj);
-            return this;
-        };
+    this.isGreater = function(value){
+      var expression = expressionFn[expressionFn.length-1];
+      if(!expression || expression.compare)
+        throw 'You must have on of the following functions ["where", "and", "or"] before use the "isGreater".';
 
-        this._match = function(obj){
-            if(expressionFn.length === 0)
-                return false;
+      expression.compare = function(propertyValue){
+        return value < propertyValue;
+      };
+      return this;
+    };
 
-            var condition = runExpression(expressionFn[0], obj);
-            for(var i=1 ; i < expressionFn.length; i++)
-                condition = runExpression(expressionFn[i], obj, condition);
+    this.isGreaterOrEqual = function(value){
+      var expression = expressionFn[expressionFn.length-1];
+      if(!expression || expression.compare)
+        throw 'You must have on of the following functions ["where", "and", "or"] before use the "isGreater".';
 
-            return condition;
-        };
+      expression.compare = function(propertyValue){
+        return value <= propertyValue;
+      };
+      return this;
+    };
 
-        this._get = function(){
-            return styles;
-        };
+    this.isLessOrEqual = function(value){
+      var expression = expressionFn[expressionFn.length-1];
+      if(!expression || expression.compare)
+        throw 'You must have on of the following functions ["where", "and", "or"] before use the "isLess".';
+
+      expression.compare = function(propertyValue){
+        return value >= propertyValue && propertyValue > 0;
```

Comment:
Why does `isLessOrEqual ` compared with 0 and the `isGreaterOrEqual` and `isGreater ` not?